### PR TITLE
Move all packets to the packet package + other stuff.

### PR DESCRIPTION
**Pull request by Blayyke**
Argo classes i found were also moved to the argo package, pending a move to their proper package + name.

### CODE REVIEW

File: mappings/net/minecraft/stat/StatsFetecherThread.mapping [hunk 1]
```diff
@@ -0,0 +1 @@
+CLASS vk net/minecraft/stat/StatsFetecherThread
```
Comment (valoeghese) [inline mappings/net/minecraft/stat/StatsFetecherThread.mapping]: Fetecher

Comment (valoeghese) [reply mappings/net/minecraft/stat/StatsFetecherThread.mapping]: Is that fetcher

Comment (shedaniel) [reply mappings/net/minecraft/stat/StatsFetecherThread.mapping]: this is 11/10 cursed mapping

File: mappings/net/minecraft/client/gui/SleepingChatScreen.mapping [hunk 2]
```diff
@@ -0,0 +1 @@
+CLASS gh net/minecraft/client/gui/SleepingChatScreen
```
Comment (shedaniel) [inline mappings/net/minecraft/client/gui/SleepingChatScreen.mapping]: Move this to `screen` sub-package

Comment (Blayyke) [reply mappings/net/minecraft/client/gui/SleepingChatScreen.mapping]: Fixed

File: mappings/net/minecraft/stat/StatsFetecherThread.mapping (hunk 1 above)
Comment (Blayyke) [reply mappings/net/minecraft/stat/StatsFetecherThread.mapping]: Fixed.

**Pull request merged**